app/workers/argo_sync_worker.rb

```diff
@@ -29,6 +29,10 @@ def perform(helm_infrastructure_id)
       EOS
     ).strip
 
+    # sleep to prevent mass sync on argo 
+    interval = 10
+    sleep interval
```

Comment:
let's move this to env var. let's use 2mins as default

Reply:
Sure. I guess 2 mins will be very long time, since we're not checking the final sync status of the app group on argo. So as far as queuing on argo is concerned, 10sec would be fine for a job?

Reply:
does 10secs enough to delay the sync in ArgoCD?

Reply:
or let's try to put this in env var, so we can adjust it right away if it's not proper yet